motor/render/vulkan_renderer.cpp

```diff
@@ -160,29 +219,6 @@ vk::Device CreateDevice(const vk::PhysicalDevice& phy_dev,
                          "Couldn't create logical device");
 }
 
-vk::CommandPool CreateCommandPool(const vk::Device& vk_dev,
-                                  size_t queue_family_idx)
-{
-  vk::CommandPoolCreateInfo pool_create_info;
-  pool_create_info.setFlags(static_cast<vk::CommandPoolCreateFlags>(0))
-      .setQueueFamilyIndex(queue_family_idx)
-      .setPNext(nullptr);
-  return VkSuccuessOrDie(vk_dev.createCommandPool(pool_create_info),
-                         "Couldn't create command pool");
-}
-
-std::vector<vk::CommandBuffer> AllocateCommandBuffers(
-    const vk::Device& vk_dev, const vk::CommandPool& cmd_pool, size_t count)
-{
-  vk::CommandBufferAllocateInfo cmd_bufer_alloc_info;
-  cmd_bufer_alloc_info.setCommandBufferCount(count)
-      .setCommandPool(cmd_pool)
-      .setLevel(vk::CommandBufferLevel::ePrimary)
-      .setPNext(nullptr);
-  return VkSuccuessOrDie(vk_dev.allocateCommandBuffers(cmd_bufer_alloc_info),
-                         "Couldn't allocate command buffers");
-}
-
 vk::SwapchainKHR CreateSwapChain(const vk::PhysicalDevice& phy_dev,
```

Comment:
again changes in this function seems isolated, could you prepare a separate change?